CMakeLists.txt

```diff
@@ -233,12 +241,17 @@ endif()
 # project that we don't actually depend on. Further some of those parts
 # do not even compile on all platforms.
 if (TORCH_MLIR_ENABLE_STABLEHLO)
-  set(STABLEHLO_BUILD_EMBEDDED ON)
-  set(STABLEHLO_ENABLE_BINDINGS_PYTHON ON)
-  add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/externals/stablehlo
-    ${CMAKE_CURRENT_BINARY_DIR}/stablehlo
-    EXCLUDE_FROM_ALL)
-  include_directories(${CMAKE_CURRENT_SOURCE_DIR}/externals/stablehlo)
+  if (NOT "${TORCH_MLIR_EXTERNAL_STABLEHLO_DIR}" STREQUAL "")
+    # Only include directories. It is assumed that stablehlo targets are made available by external project.
+    include_directories(${TORCH_MLIR_EXTERNAL_STABLEHLO_DIR})
```

Comment:
I would have assumed that the external project also makes the includes available.